boot.js

```diff
@@ -207,16 +205,19 @@ function callWithCbOrNextTick (func, cb, context) {
     context = this._server
   }
 
+  var err = this._error
```

Comment:
I think you should wrap this inside an `if (this)`, as was done few lines above :)
Or, if not needed, remove the `if (this)` few lines above :P